src/win32/kerberos_win32.cc

```diff
@@ -127,14 +128,12 @@ NAN_METHOD(KerberosClient::WrapData) {
 KerberosServer::~KerberosServer() {
     // if (_state != NULL) {
     //     authenticate_gss_server_clean(_state);
+    //     free(_state);
     //     _state = NULL;
     // }
```

Comment:
This is unused right since we don't use CPP automatic destruction? I guess this is fine to leave commented, it goes along with the other TODOs.

Reply:
It’s unused because `KerberosServer` is unimplemented on Windows. I just added this as a reminder to actually clean up everything if support for it is eventually added at some point.

Reply:
> This is unused right since we don't use CPP automatic destruction?

Also, for context – we do use it implicitly by using `Nan::ObjectWrap`, which calls `delete object;` once the garbage collector collects the corresponding JS object.